### ISSUE TITLE
训练gme发现警告，我用的是最新的版本

### ISSUE THREAD

**Issue by Liyunlong24** [modelscope/ms-swift#9709]
### Checklist / 检查清单

- [x] I have searched existing issues, and this is a new bug report. / 我已经搜索过现有的 issues，确认这是一个新的 bug report。

### Bug Description / Bug 描述

No label_names provided for model class `PeftModel`. Since `PeftModel` hides base models input arguments, if label_names is not given, label_names can't be set automatically within `Trainer`. Note that empty label_names list will be used instead.
`loss_type=None` was set in the config but it is unrecognised.Using the default loss: `ForCausalLMLoss`.

### How to Reproduce / 如何复现

nproc_per_node=8

# losses: swift/loss
# 8*40G
MAX_PIXELS=65536 \
NPROC_PER_NODE=$nproc_per_node \
swift sft \
    --model iic/gme-Qwen2-VL-2B-Instruct \
    --tuner_type lora \
    --dataset 'swift/TextCaps:emb' \
    --load_from_cache_file true \
    --split_dataset_ratio 0.01 \
    --torch_dtype bfloat16 \
    --num_train_epochs 1 \
    --per_device_train_batch_size 2 \
    --per_device_eval_batch_size 2 \
    --gradient_accumulation_steps $(expr 64 / $nproc_per_node) \
    --eval_steps 100 \
    --save_steps 100 \
    --eval_strategy steps \
    --save_total_limit 2 \
    --logging_steps 5 \
    --output_dir output \
    --lazy_tokenize true \
    --warmup_ratio 0.05 \
    --learning_rate 5e-5 \
    --deepspeed zero3 \
    --dataloader_num_workers 4 \
    --task_type embedding \
    --loss_type infonce \
    --dataloader_drop_last true

**Comment by tastelikefeet** [comment]
loss 正常吗
如果正常这个 warning 可以忽略

**Comment by Liyunlong24** [comment]
> loss 正常吗 如果正常这个 warning 可以忽略

感谢回复，loss看起来是正常的
